### ISSUE TITLE
Adjust routing to make react UI primary UI

### ISSUE THREAD

**Issue by kitsunet** [neos/neos-ui#1290]
Decide on `legacy` backend route (`/neos-legacy`?)
Switch routes
Test
Adjust loading order

**Comment by kitsunet** [comment]
From @skurfuerst in Slack:

FYI: over the weekend some ideas regarding how to "make the switch" from the old Ember UI to the new React UI. Before, it was still somewhat vague for me, but now that I had some time to think it through, I think I came up with a good and easy to   implement solution.

I propose the following:

1) The Neos React UI should add routes for /neos and /neos!. We furthermore configure Neos.Neos.Ui routes to be added *before* Neos.Neos Routes.
2) Furthermore, we add a transitional, unsupported setting "enableOldUi"; set it by default to "FALSE" -> if this setting is set to FALSE, the service which determines whether the old or new UI is enabled will always return the NEW UI.
3) this way, if people install the package, it will *replace* the old UI by the new one.
4) we're having some bigger corporate customers who need/want to extend the transition period some more and we'd help them a lot if we were able to keep the current behavior which we have now; but I would clearly state it is UNSUPPORTED. To get this behavior back, one would need to configure the Routes ordering differently, so that Neos.Neos is loaded before Neos.Neos.UI; and the "enableOldUi" switch needs to be set to TRUE.
